include/base.php

```diff
@@ -62,7 +62,7 @@ public function __construct( &$links_model ) {
 		add_action( 'pll_language_defined', array( $this, 'load_strings_translations' ), 5 );
 		add_action( 'change_locale', array( $this, 'load_strings_translations' ) ); // Since WP 4.7
 		add_action( 'personal_options_update', array( $this, 'load_strings_translations' ), 1, 0 ); // Before WP, for confirmation request when changing the user email.
-
+		add_action( 'lostpassword_post', array( $this, 'load_strings_translations' ), 1, 0 ); // Before WordPress switch the locale in retrieve_password().
```

Comment:
```suggestion
		add_action( 'lostpassword_post', array( $this, 'load_strings_translations' ), 10, 0 ); // Password reset email.
```
In this case, we don't need use a specific priority, so we can keep the default value. The second parameter set to 0 is important though.